flang/test/Driver/linker-flags.f90

```diff
@@ -21,12 +22,12 @@
 !       run on any other platform, such as Windows that use a .exe
 !       suffix. Clang's driver will try to resolve the path to the ld
 !       executable and may find the GNU linker from MinGW or Cygwin.
-! GNU-LABEL:  "{{.*}}ld{{(\.exe)?}}"
-! GNU-SAME: "[[object_file]]"
-! GNU-SAME: -lFortran_main
-! GNU-SAME: -lFortranRuntime
-! GNU-SAME: -lFortranDecimal
-! GNU-SAME: -lm
+! UNIX-LABEL:  "{{.*}}ld{{(\.exe)?}}"
+! UNIX-SAME: "[[object_file]]"
+! UNIX-SAME: -lFortran_main
```

Comment:
Personally I prefer testing `-lx -ly -lz` on one line to assert that is no extra argument in between.
This line isn't very long.

Reply:
Right: they fit nicely even with the double quotes.  To avoid swamping the patch with unrelated changes, I've only adjusted the `UNIX` case.